packages/widgetbook/README.md

```diff
@@ -11,22 +11,36 @@ A flutter package which helps developers cataloguing their widgets, test them qu
 <img src="https://media.githubusercontent.com/media/widgetbook/widgetbook/main/docs/assets/Screenshot.png" alt="Widgetbook Screenshot" />
```

Comment:
Please edit the first sentence. It was my fault: A flutter package which helps developers cataloguing their widgets, testing them quickly on multiple devices and themes, and sharing them easily with designers and clients. Inspired by Storybook.js and flutterbook.

Mistakes were at test and share. Ing was forgotten